client/components/jetpack/jetpack-disconnected-wpcom/index.tsx

```diff
@@ -20,27 +19,23 @@ const JetpackDisconnectedWPCOM: FunctionComponent = () => {
 	const onSupportClick = useTrackCallback( undefined, 'calypso_jetpack_backup_support_click' );
 	return (
 		<PromoCard
-			title={ preventWidows( translate( 'Jetpack connection has failed' ) ) }
+			title={ translate( 'Jetpack connection has failed' ) }
 			image={ { path: JetpackDisconnected } }
 			isPrimary
 		>
 			<p>
-				{ preventWidows(
-					translate( 'Jetpack is unable to reach your site {{siteName/}} at this moment.', {
-						components: { siteName: <strong>{ siteName }</strong> },
-					} )
-				) }
+				{ translate( 'Jetpack is unable to reach your site {{siteName/}} at this moment.', {
+					components: { siteName: <strong>{ siteName }</strong> },
+				} ) }
 			</p>
 			<p>
-				{ preventWidows(
-					translate(
-						'Please visit {{siteUrl/}} to ensure your site is loading correctly and reconnect Jetpack if necessary.',
-						{
-							components: {
-								siteUrl: <ExternalLink href={ siteUrl }>{ siteUrl }</ExternalLink>,
-							},
-						}
-					)
```

Comment:
Not sure how to test this one but internal text is rendering inside `p` tags and PromoCard `title` goes into a `h2`.